baker/batchTagWithGpt.ts

```diff
@@ -0,0 +1,88 @@
+import * as db from "../db/db.js"
+import { Chart } from "../db/model/Chart.js"
+import yargs from "yargs"
+import { hideBin } from "yargs/helpers"
+
+interface BatchTagWithGptArgs {
+    debug?: boolean
+    limit?: number
+}
+
+export const batchTagWithGpt = async ({
```

Comment:
Presumably wrapped because we might be batch tagging other things in the future?

Reply:
Exactly